infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackController.kt

```diff
@@ -265,4 +246,18 @@ class LocationTrackController(
         logger.apiCall("getLocationTrackOwners")
         return locationTrackService.getLocationTrackOwners()
     }
+
+    @PreAuthorize(AUTH_ALL_READ)
+    @GetMapping("/{publishType}/{id}/splitting-initialization-parameters")
+    fun getSplittingInitializationParameters(
+        @PathVariable("publishType") publishType: PublishType,
+        @PathVariable("id") id: IntId<LocationTrack>,
+    ): SplittingInitializationParameters {
+        logger.apiCall(
+            "getSplittingInitializationParameters",
+            "publishType" to publishType,
+            "id" to id,
+        )
+        return locationTrackService.getSplittingInitializationParameters(id, publishType)
+    }
```

Comment:
Groupattu kaikki pilkkomisen tarvitseman alustusdatan (vaihteet raiteella, raiteen duplikaatit) haku yhden endpointin taakse. Näistä tarvitaan hitusen keskimääräistä enemmän tietoa (tarvitaan mm. vaihteiden sijaintitiedot sekä duplikaattiraiteiden alku- ja loppusijainnit sekä m-arvot,) joten näitä tarpeita varten tehty omat, koostavat tietotyypit